mfaliquot/application/updater.py

```diff
@@ -283,6 +287,13 @@ def postloop_finalize(self, terminated):
           merges = self.seqinfo.find_and_drop_merges()
           if not merges:
                _logger.info("No merges found")
+          else:
+               _logger.warning(f"Writing merges to {self.mergefile}")
+               for target, drops in merges:
+                    with open(self.mergefile, 'a') as f:
```

Comment:
The `open` call should be outside the loop, no reason to do a syscall `n` times instead of 1 time

(but don't worry about it, I'll make my own commit to fix it)